ignite/metrics/gan/fid.py

```diff
@@ -93,7 +75,7 @@ class FID(Metric):
         num_features: number of features, must be defined if the parameter ``feature_extractor`` is also defined.
             Otherwise, default value is 2048.
         feature_extractor: a callable for extracting the features from the input data. If neither num_features nor
-            feature_extractor are defined, default value is ``InceptionExtractor``.
+            feature_extractor are defined, default value is ``InceptionModel``.
```

Comment:
If you mention `InceptionModel` as a class name in the docs, you have to add `InceptionModel` into the docs and add `InceptionModel` as a link.

Reply:
```suggestion
            ``feature_extractor`` are defined, default value is ``InceptionModel``.
```